src/tracks/playabletrack/wavetrack/ui/WaveTrackMenuItems.cpp

```diff
@@ -59,16 +72,31 @@ void OnNewStereoTrack(const CommandContext &context)
 
    SelectUtilities::SelectNone( project );
 
-   tracks.Add(trackFactory.Create(2, defaultFormat, rate));
-   auto &newTrack = **tracks.rbegin();
-   newTrack.SetSelected(true);
-   newTrack.SetName(tracks.MakeUniqueTrackName(WaveTrack::GetDefaultAudioTrackNamePreference()));
+   auto track = trackFactory.Create(2, defaultFormat, rate);
+
+   wxString newTrackPlacement = gPrefs->Read(wxT("/Tracks/NewTrackPlacement"), wxT("BottomOfProject"));
+
+   if (newTrackPlacement == wxT("TopOfProject")) {
+      tracks.Insert(*tracks.begin(), track);
+   } else if (newTrackPlacement == wxT("AboveCurrentTrack")) {
+      auto focusedTrack = TrackFocus::Get(project).Get();
+      tracks.Insert(focusedTrack, track);
+   } else if (newTrackPlacement == wxT("BelowCurrentTrack")) {
+      auto focusedTrack = TrackFocus::Get(project).Get();
+      tracks.Insert(tracks.GetNext(*focusedTrack), track);
+   } else {
+      tracks.Add(track);
+   }
```

Comment:
Similar block of code repeats 3 times - it can be one function used across these 3 places

Reply:
@grliszas14 , do you have a suggestion where this function could be placed? I was thinking on `WaveTrackUtilities`, or even on `Tracks.cpp` but here I was not able to import `TrackFocus`.